play-media-native/components/FacetFilters/FacetFilters.tsx

```diff
@@ -37,6 +42,11 @@ export const FacetFilters = ({ facetFilters, onChange }: FacetFiltersProps) => {
             paddingHorizontal: theme.spacing.xxs,
             flexShrink: 1,
           }}
+          selectedValue={
+            type === CONTENT_TYPES.EVENT
+              ? eventFilterValues?.[facet.id]
+              : athleteFilterValues?.[facet.id]
+          }
```

Comment:
FacetFilters is a reusable component. Think of scaling. More entities to filter means more conditionals in here. Instead of having conditionals inside the reusable component, pass the selected value from outside to keep the reusable component clean.

Reply:
FacetFilters is indeed a reusable component, however it is used only for some very specific use cases (event filtering, athlete filtering and media filtering when available). Having the selected value being passed from the outside means we would have to retrieve the store values in more places, leading to more code, harder debugging etc. Can't say I see any added value here, only complexity.

Reply:
If you were a Sitecore CH ONE client wanting to use some of our code, would you want this pluggable element to contain events and athletes if you don't care about events and athletes?

Reply:
There is incredible value in assisting our clients to get their job done easier.

Reply:
No client is going to use these assets for their own purposes. This is not a starter kit.